UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -168,6 +177,11 @@
   MSFT:RELEASE_*_*_CC_FLAGS      = /D MDEPKG_NDEBUG
 !endif
 
+!if $(DISABLE_MMX_SSE) == TRUE
+[BuildOptions.X64, BuildOptions.IA32]
+  GCC:*_*_*_CC_FLAGS         = -mno-mmx -mno-sse

```

Comment:
Prefer you can add new build option "--add_cc_flags" into UniversalPayloadBuild.py 

Usage:

UniversalPayloadBuild.py

py -3 UniversalPayloadBuild.py --add_cc_flags=-mno-mmx --add_cc_flags=-mon-sse
---> build.py -DAPPEND_CC_FLAGS="-mno-mmx -mon-sse"

UefiPayloadPkg.dsc
*_*_*_CC_FLAGS = $(APPEND_CC_FLAGS)

Reply:
Get you. Let me follow your instructions to refine and validate it. Then update this commit. Thanks.